lib/src/checkbox/Checkbox.tsx

```diff
@@ -174,7 +177,7 @@ const CheckboxContainer = styled.span`
   align-items: center;
   cursor: ${(props) => (props.disabled ? "not-allowed" : "pointer")};
   position: relative;
-  flex-direction: ${(props) => (props.labelPosition === "before" ? "row-reverse" : "row")};
+  vertical-align: top;
   .MuiCheckbox-colorSecondary {
```

Comment:
I see that the visual regression tests show a change in the alignment or height of the component, but I have not changed any styles that affect it.

Reply:
The height of the component remains the same before and after these changes (24px), looking at the chromatic story it seems to be an issue with the heading rather than the checkbox, which is pushing elements down progressively.